raiden-dapp/src/views/account/UDC.vue

```diff
@@ -1,133 +1,52 @@
     </v-row>
-    <v-row justify="center">
-      <v-col cols="10">
-        <h3 class="udc__sub-head">
-          {{ $t(mainnet ? 'udc.deposit' : 'udc.mint-deposit') }}
-        </h3>
-      </v-col>
+    <v-row class="udc__balance" no-gutters justify="center">
+      {{ $t('udc.balance') }}
+      <amount-display
+        class="udc__balance__amount"
+        inline
+        :amount="udcCapacity"
+        :token="udcToken"
+      />
     </v-row>
-    <v-row v-if="!loading && !error" justify="center" no-gutters>
-      <v-col cols="6">
-        <v-text-field
-          v-model="amount"
-          autofocus
-          type="text"
-          :suffix="serviceToken"
-          class="udc__mint-deposit--amount"
+    <v-row class="udc__actions" no-gutters justify="center">
+      <v-btn text class="udc__actions__button" @click="showUdcDeposit = true">
+        <v-img
+          width="70px"
+          height="65px"
+          :src="require('../../assets/deposit.svg')"
         />
-      </v-col>
-    </v-row>
-    <v-row v-else-if="error" justify="center">
-      <v-col cols="10">
-        <error-message :error="error" />
-      </v-col>
-    </v-row>
-    <v-row v-else class="udc__progress" justify="center">
-      <v-col cols="10">
-        <v-row>
-          <spinner />
-        </v-row>
-        <v-row no-gutters align="center" justify="center">
-          <span v-if="step === 'mint'">
-            {{
-              $t('udc-deposit-dialog.progress.mint', {
-                currency: serviceToken,
-              })
-            }}
-          </span>
-          <span v-else-if="step === 'approve'">
-            {{
-              $t('udc-deposit-dialog.progress.approve', {
-                currency: serviceToken,
-              })
-            }}
-          </span>
-          <span v-else-if="step === 'deposit'">
-            {{
-              $t('udc-deposit-dialog.progress.deposit', {
-                currency: serviceToken,
-              })
-            }}
-          </span>
-        </v-row>
-      </v-col>
-    </v-row>
-    <v-row v-if="!loading && !error" justify="center">
-      <v-col cols="10" class="text-center">
-        <p>
-          {{
-            $t('udc-deposit-dialog.available', {
-              balance: accountBalance,
-              currency: $t('app-header.currency'),
-            })
-          }}
-        </p>
-      </v-col>
-    </v-row>
-    <v-row v-if="!loading && !error" justify="center">
-      <action-button
-        arrow
-        :enabled="valid && !loading"
-        :text="
-          $t(
-            mainnet
-              ? 'udc-deposit-dialog.button-main'
-              : 'udc-deposit-dialog.button'
-          )
-        "
-        class="udc__action"
-        @click="udcDeposit()"
-      />
+        <span class="udc__actions__button--title">
+          {{ $t('udc.deposit') }}
+        </span>
+      </v-btn>
+      <v-btn text class="udc__actions__button" @click="withdrawFromUdc = true">
```

Comment:
Nitpick:
The button above has an action called `showUdcDeposit`. I would expect this do be similar. So I would either decide to say that we want to show a dialog or what the dialog its purpose is (e.g. deposit to udc).

Reply:
I just kept the same variable name as in the other components where it is used, makes it a bit easier when searching I think.